widget/button.go

```diff
@@ -125,8 +125,11 @@ func (b *buttonRenderer) Refresh() {
 		b.icon.Hidden = true
 	}
 
+	b.label.Refresh()
+	b.icon.Refresh()
+	b.shadow.Refresh()
+
 	b.Layout(b.button.Size())
-	canvas.Refresh(b.button)
```

Comment:
I would change this one back. The `canvas.Refresh` will repaint the sub-items.
I also think that's a better approach to use `Hide()`, `Show()` or whatever state-changing method, which implicitly refreshes if necessary, over `Hidden = … ` + `Refresh()`.